### ISSUE TITLE
Extremely low Specific intestinal permeability values calculated with PK-Sim

### ISSUE THREAD

**Issue by TatianaAlieva** [Open-Systems-Pharmacology/Forum#23]
Hello to everybody,
I tried permeability prediction in PK-Sim for three neutral compounds from the paper Thelen et al., 2011: theophylline, paracetamol, caffeine. The resulting permeabilities displayed in the “Specific intestinal permeability” of ADME tab are 2-3 orders of magnitude below Caco-2/MDCK permeability values (see the attached file). What could be the reason for that?
Thank you in advance!
[Low_Permeability.pdf](https://github.com/Open-Systems-Pharmacology/Forum/files/1069622/Low_Permeability.pdf)

**Comment by Yuri05** [comment]
@TatianaAlieva Could you please also attach your PK-Sim project (you must zip it first)?

**Comment by TatianaAlieva** [comment]
Excuse me, I've zipped the project, but github still does not recognize it as a supported  file-type :(

**Comment by Yuri05** [comment]
Please check this: https://help.github.com/articles/file-attachments-on-issues-and-pull-requests/ 
If attaching per Drag&Drop does not work, click on "**selecting them**"

**Comment by TatianaAlieva** [comment]
I have tried different archivers and various types of archives, the github still does not recognise files as a supported file-type. May I send the file through e-mail?

**Comment by msevestre** [comment]
It might be an issue with the Webbrowser you are using. I have experienced similar problems with an outdated version of Firefox

**Comment by TatianaAlieva** [comment]
i have checked the version of Firefox, it is the latest.

I was able to attach pdf to the first message. It is an issue with attachment of archived file.

**Comment by msevestre** [comment]
@TatianaAlieva Would it be possible to upload that file somewhere (drive, dropbox etc..) and paste a link here? We could upload on your behalf

**Comment by TatianaAlieva** [comment]
https://drive.google.com/open?id=0B5PLXyVor3O3WG42MERBUEpGSmc

**Comment by msevestre** [comment]
[Permeability_Prediction.zip](https://github.com/Open-Systems-Pharmacology/Forum/files/1071750/Permeability_Prediction.zip)

Uploaded

**Comment by TatianaAlieva** [comment]
I have tried permeability prediction for some 15 compounds. It looks like there is just a mistake in units: to my understanding, if they are meant cm/s instead of cm/min, the values become reasonable.

**Comment by msevestre** [comment]
@TatianaAlieva Mistake in units is extremely unlikely for such a central parameter.

**Comment by TatianaAlieva** [comment]
Hello,
I have calculated the intestinal permeability for 25 compounds in PK-Sim.
Almost all values are unrealistically low. Fig. 1A shows them plotted against cell-monolayer permeability. Predicted by PK-Sim permeability values are up to 3,5 orders below cell-monolayer permeability.
However if to assume that calculated by PK-Sim values are in cm/s units (instead of cm/min as is given in the interface), then the values are ok (Fig. 1B).
Excuse me, are you sure that permeability units in PK-Sim interface are correct?
[Fig1.pdf](https://github.com/Open-Systems-Pharmacology/Forum/files/1077080/Fig1.pdf)

**Comment by TheiBa** [comment]
@TatianaAlieva,

I think PK-Sim P_int is not 1to1 P_eff. For Caco/Pampa we used a lab specific calibration/mapping (coz of the high inter-lab variability), that was license-specific and thus deactivated during source-opening. We will look for more infos next week - it is public holidays / long weekend in (most of) Germany ...


Von meinem iPhone gesendet

**Comment by Aedginto** [comment]
Hi Tatiana,
The intestinal permeability parameter in PK-Sim is estimated using an algorithm that accounts for molecular size and lipophilicity. Whenever an algorithm is used, there is uncertainty.  Further, any intestinal permeability metric that is measured experimentally is lab specific. Caco2 is good for looking at relative permeability x surface area products (called effective permeability) within one lab but absolute values are not necessarily transferable between labs. This is why one must calibrate the lab's Caco2 outcomes to the algorithm (it is not a unit issue) and use the same scaling factor every time. From the figure that you attached, a scaling factor could easily be optimized. 
All the best!
Andrea

**Comment by TatianaAlieva** [comment]
Hi,
Thank you for comments!

**1. As of interlaboratory variation in cell-based assays:**
Statistical analysis of 441 individual Caco-2 and MDCK permeability measurements from 55 independent studies showed that **the average standard deviation** (SD) of the **passive transcellular** permeability component due to interlaboratory variation is **0,35 log units** (Avdeef A, Absorption and Drug Development: Solubility, Permeability, and Charge State, 2012, p. 561) ). SD above 0,6 log units is really rear (see Table 8.6 from this book). So, interlaboratory variation in cell-monolayer assays is unlikely to explain permeability shift above ~0,5 log units.
In contrast, the shift in the figure attached earlier was ~1,5-2 log units.

**2. Concerning apparent (or effective) permeability.** I did not compare predictions to it.
The earlier presented figure shows calculated in PK-Sim values plotted against **passive transcellular** component of Caco-2/MDCK permeability  listed in Table 8.6 “Absorption and Drug Development: Solubility, Permeability, and Charge State”. Moreover, this passive transcellular permeability was **averaged** over reports from different laboratories (if several were available).
In contrast to the **apparent (effective) permeability** measured in experiments, **passive transcellular** permeability excludes contributions of the hydrodynamic resistance of UWL, paracellular permeability, active transport (e.g. Adson et al., 1995; Avdeef, 2012). In addition, averaging minimizes the impact of the laboratory specific variation.

**3.** In general, there is a growing body of evidence suggesting that the passive transcellular permeability of Caco-2 and MDCK cell monolayers reproduces the transcellular permeability of the enterocytes layer in the intestine and endotheliocytes layer in the blood-brain barrier in 1:1 ratio (Avdeef and Tam, 2010; Avdeef, 2011; Yusof et al., 2014). So, the correction factor should not emerge for the intrinsic difference between the passive transcellular permeability of cell-monolayers and the intestinal epithelium.

**4.** @Aedginto 
> Caco2 is good for looking at relative permeability x surface area products (called effective permeability)

Can you, please, give the reference that uses the effective Caco-2 permeability term in this meaning?
The following definition is taken from Sun et al., 2008:
“The apparent or effective permeability (P app ), the parameter describing the flux at which the molecule traverses **per unit area of the cell barrier**”.
See also Eq. 1 from Karlsson and Artursson, 1991; Eq.1 from Adson et al., 1995; P.2117  **NATURE PROTOCOLS**, vol.2 no.9, 2007, and so on.
In fact, I cannot find a study over 30+ papers that would use effective permeability term as a product of permeability x surface area. 
According to Adson et al., 1995; Avdeef 2012, and many others the effective permeability coefficient differs from transcellular permeability in that that it includes the mitigating effect of UWL, filter support, contribution from paracellular permeability. However all of them are meant per unit cel monolayer surface area.
There are techniques developed to separate these contributions (summarized e.g. in Avdeef, 2012). As I stated above, I plotted PK-Sim predictions against passive transcellular permeability component.

So, I am still wondering what can cause 1,5-2 log units permeability shift.

P.S. It is possible to introduce an _ad hoc_ correction factor without much thinking about, however without understanding what does it reflect, it undermines the principles of the mechanistic modeling. Therefore I am trying to understand what the physical meaning of this shift is.

**Comment by Aedginto** [comment]
Thanks for the interesting discussion! I have a few comments.

I think it is important to state that the apparent permeability from a Caco-2 assay is not directly translatable to the in vivo permeability in a mechanistic absorption model without the use of scaling factors, unless the mechanistic absorption model was empirically built to use a lab-specific Caco-2 output. I will explain as I go. 

First, as you have identified, there is inter-lab variability. Regardless of the value of that variability, it represents a component of a scaling factor that should be implemented in a lab-specific mechanistic absorption model. 

Second, the area in a Caco-2 system is that of the monolayer. Not the absorptive surface area in the system, which is unknown. As far as I know (would be interested to hear others on this though!), the brush border is intact and unaccounted for in the area value. This is why I stated that the outcome of a Caco-2 assay is permeability *surface area. That is the known value from that assay. Dividing by area that is not necessarily the actual surface area gives you an apparent permeability, not a permeability. As such, a scaling factor that accounts for this uncertainty will be required to transfer to a mechanistic model. 

Third, as you already wrote, Caco-2 accounts for most of the transport processes that a molecule undergoes. I am unsure of how a passive transcellular permeability is absolutely ensured from a Caco-2 assay; I suspect with the use of active transport inhibitors(?). The PAMPA system may be better for estimating passive permeability. 

On to the PK-Sim algorithm. Its development is described in Willmann et al.  A physiological model for the estimation of the fraction dose absorbed in humans. J Med Chem. 2004 Jul 29;47(16):4022-31. It is important to note that the Pint algorithm, which uses a measure of lipophilicity and molecular volume, is empirically derived within a mechanistic gut model framework. As such, while there was an extensive list of drugs for which fraction absorbed was known and simulated, the resulting Pint is then only valid for that model structure and parameterization from which it was derived. In short, the Pint from PK-Sim only makes sense in PK-Sim.

What I have convinced myself of is that there is no method that I know of to measure the true Pint. It is unknown and not precisely knowable. The P that is needed in a mechanistic absorption model is specific to the model structure and parameterization. If the model structure or parameter values change, the required Pint to get a good ‘fit’ (assuming sensitivity) may be different than it was before the change. Intestinal permeability in a mechanistic absorption model is an uncertain parameter unless benchmarked (scaled) with some type of ‘real’ data. 

Nor is there a true mechanistic absorption model. They all have different structures and parameters that may or may not be better or worse for a specific scenario.  

In summary, a deviation between a model derived Pint and an experimental Pint are both different ways of thinking about true Pint, but they are not comparable. 

For your purposes, I understand that a scaling factor is unmechanistic…I completely agree but in the absence of the ability to determine true Pint, I don’t see another way. 

Let me know your thoughts:-)
Take care,
Andrea

**Comment by TatianaAlieva** [comment]
Hi to all,
Andrea, really many thanks for your comments! Excuse me, for keeping silence, I took time to think of them.

But first, as I go in depth, new questions arise. Since neither Andrea nor me have developed the PK-Sim absorption model, comments of the authors of PK-Sim intestinal absorption model would be helpful:
1. I have plotted permeability values from Supplementary materials of Willmann et al., 2004 (J Med Chem. 2004 Jul 29;47(16):4022-31) against PK-Sim 2017 values for 21 out of 25 compounds from Fig 1 presented in this thread earlier. Over half of the values predicted by PK-Sim are 10-300 times smaller than Willmann (2004) values. Could you, please, explain the reason behind the range shift in the predicted permeability?
![fig 2](https://user-images.githubusercontent.com/29387078/27589797-fe6e17c2-5b4c-11e7-8b6c-a8c7a025f643.png)

Values from Willmann et al. (2004) were in surprisingly good agreement with the transcellular permeability of Caco-2/MDCK cell monolayers taking into account presence (Fig .3).
![fig 3](https://user-images.githubusercontent.com/29387078/27589835-159e2766-5b4d-11e7-9696-4456b337af21.png)
Leveling above “-5”  corresponds to UWL-limited permeability regime present in Willmann et al. model, if i am not mistaken.
2. To better understand: what surface area (that of enterocytes apical membrane or of the intestinal epithelium) do permeability values in (a) PK-Sim and (b) Supplementary material to Willmann et al (2004) refer to?

**Comment by Aedginto** [comment]
Hi again!
There was an update to the Willmann 2004 model. Because the model structure and parameterization had changed, a new empirical fitting process was done. The ref for this is: "Evolution of a detailed physiological model to simulate the gastrointestinal transit and absorption process in humans, Part 1: Oral solutions." So sorry to have not mentioned the update in my last post!

With regards to surface area, this would be the gut surface area which incorporates the 2 dimensional surface of each gut section times enhancement factors that account for folds, villi and microvilli. The enhancement factors may be mentioned in the above reference.

Take care,
Andrea

**Comment by TatianaAlieva** [comment]
Now, to Andrea’s comments: 

> On to the PK-Sim algorithm. Its development is described in Willmann et al. A physiological model for the estimation of the fraction dose absorbed in humans. J Med Chem. 2004 Jul 29;47(16):4022-31. It is important to note that the Pint algorithm, which uses a measure of lipophilicity and molecular volume, is empirically derived within a mechanistic gut model framework. As such, while there was an extensive list of drugs for which fraction absorbed was known and simulated, the resulting Pint is then only valid for that model structure and parameterization from which it was derived. In short, the Pint from PK-Sim only makes sense in PK-Sim.

I see your point, but cannot understand why you find Pint in PK-Sim model (Willmann et al., 2004) to be defined in some model-specific way? From the paper I can see that permeability is introduced by Eq. 6 consistently with the commonly-used definition of the intestinal permeability. The absorption (eq. 8) is calculated rigorously mechanistically. The way how permeability is predicted from compounds properties has no influence on this.
Besides, permeability values to parametrize permeability equation were calculated by fitting the mechanistic absorption integral (Eq.8, Willmann et al., 2004) to real FA values using realistic estimates of the intestinal surface area and the drug residence time. Therefore, I expect the derived permeability values to be realistic and permeability predictions from trained on such dataset permeability equation 9 (Willmann et al., 2004) to be of correct order of magnitude.
In contrast, the calculated in PK-Sim permeability values are on average ~60 times lower. The interlab variation covers 2-fold uncertainty. Where does the rest shift of 30 times come from?

Further, I think there is much misunderstanding due to messy permeability nomenclature :(. There are several permeability scales in parallel use.
1)	The established practice in cell-based assays is always to refer to the unit surface of the cell monolayer. This method is also often used to define the intestinal permeability. All that I wrote about permeability in my previous posts refer to this scale.
2)	_In situ_ effective permeability measured in humans is presented in terms of the “smooth-tube” model of the intestine, i.e. as if there are no circular folds, villi and microvilli in the intestine (e.g. Dahlgren et al., 2015).
I do not comment is it right or wrong to relate permeability according to the first or second definition. But it is a convention to report values in either of these two scales accepted in all permeability literature I have ever read.
3)	If I correctly understand you, Andrea, you refer the true permeability to the absorptive surface area of the enterocytes and Caco-2 cells, aren’t you?
To my knowledge, it is a very unusual way to express permeability of epithelium monolayers. I have never come across such permeability scale in any paper concerning permeability of cell-monolayers _in vitro_ and the intestinal barrier _in vivo_ before.
Moreover, I am unsure that it is correct to relate permeability to the absorptive surface area of the cell layer. I wonder what you and others would say about the following reasoning. Please, look at this picture.
 ![flux](https://user-images.githubusercontent.com/29387078/27659142-b70414fc-5c52-11e7-9e39-637008ca4cc2.png)
 First of all, technically, permeation of the barrier is proportional to its **cross-sectional** area, not its **absorptive surface** area (It’s a**D**sorption that is proportional to the surface area). If the depicted cylinder had an uneven left base, this would not change the flux through it.
If a barrier consists of only one membrane, then its cross-sectional area is identical to its surface area. In contrast, the diffusional barrier of a cell-monolayer is set by the apical and basolateral membranes in series. Moreover, these membranes are of different surface areas. Technically, we have a barrier of the variable cross-section. What surface area to relate the flux to? (i.e. which surface area can serve as the effective cross-sectional area).
Permeability, P, arises as a proportionality factor in the description of mass-transport:
dQ/dt=P*S*gradC (1),
where dQ/dt – the flux across the cross-sectional area S, gradC –conc. gradient.
It follows from this equation if the cross-sectional area is n-fold higher, the flux dQ/dt is also n-fold higher.
If we set the effective cross-sectional area of the cell barrier to be equal to the absorptive surface area (i.e. we normalize drug flux across the cell-monolayer to it), we will run into the following trouble.
Consider two cell layers: one with smooth basal and apical membranes, the other with microvilli on the apical surface, which increase the absorptive surface area n-fold. Will the flux across “microvilli” cell layer be n-fold higher, as follows from the mass-transport equation?
No, because the diffusion across the basolateral membrane becomes a rate-limiting step in the “microvilli” cells. This is in contradiction with the above-mentioned consequence of the mass-transport equation.
Use of the secretive surface area of the cell-monolayer as an effective cross-sectional area has the same difficulties. Only the surface area of the monolayer as it takes on the filter support escapes from this difficulty 
What do you think about this?

**Comment by TatianaAlieva** [comment]
> There was an update to the Willmann 2004 model. Because the model structure and parameterization had changed, 
Could you be, please, more specific?

**Comment by TatianaAlieva** [comment]
Continuing about permeability scales.
To go from one permeability scale to other only the respective surface expansion factor is required and they are well-established and consistent with those of PK-Sim model as I can see.

Further, the transcellular permeability component of Caco-2 and MDCK was shown to reflect closely to 1:1 the corresponding component of human jejunal in vivo permeability when both are expressed according to permeability scale №1 (Avdeef and Tam, 2010; Sjorgren et al., 2013). Since this is experimentally established, the empiric factor to scale Caco-2 transcellular permeability to the intestinal permeability has to be no higher than the interlaboratory variation (0,35 log units).

Now, you, Andrea, wrote:

> I think it is important to state that the apparent permeability from a Caco-2 assay is not directly translatable to the in vivo permeability in a mechanistic absorption model without the use of scaling factors, unless the mechanistic absorption model was empirically built to use a lab-specific Caco-2 output. 
1)	I agree that if permeability values are expressed in different scales, then a scaling factor is necessary. My point is that this factor is not a freely adjustable parameter: at least, its order of magnitude has to be consistent with the respective surface area expansion factor.
2)	To specify: I am not talking about any arbitrary taken Caco-2 permeability dataset. I am talking (and using) accurately determined by A. Avdeef (2012) transcellular components of Caco-2 or MDCK cell monolayers permeability. To explain the difference: there are a lot of permeability studies with inefficient stirring or leaky tight junctions where respective permeability components obscure the true transcellular permeability. Such Caco-2 or MDCK data have limited predictive power for the intestinal absorption and I do not refer to it.
> What I have convinced myself of is that there is no method that I know of to measure the true Pint. It is unknown and not precisely knowable. 

Definitely I am not talking about precise values. Nevertheless these values are measured with the acceptable accuracy. E.g. Avdeef and Tam, 2010; Dahlgren et al., 2015.

**Comment by Aedginto** [comment]
The 2004 paper describing the PK-Sim gut model had a continuous tube structure and parameterization. The revised version is more of an ACAT-like model including the mucosa (Figure 2 in the 2011 paper). As stated in the abstract of the 2011 paper, “A training set of 111 passively absorbed drugs with reported
fractions of dose absorbed was used to optimize the semiempirical equation, which calculates
intestinal permeability coefficients.” The result of any optimization is a function of the model structure. Once Pint was optimized within the model for fraction absorbed of a bunch of compounds, those Pint values were regressed with a lipophilicity and size metric. What comes of this is a way of calculating Pint based on a compounds lipo and size. The way that Pint is then used in the equations to estimate rate of absorption in the model are typical such that Pint is multiplied by gut surface area. 

Pint (cm/s) and surface area (cm^2) are unique and independent parameters in any equation of mass transfer across a membrane. They do not depend on one another. When I speak of Pint, there is no surface area attached to it because the surface area has nothing to do with Pint. In real life however, uniquely identifying intestinal Pint and intestinal surface area is not possible because the output – flux across a membrane – is dependent on Pint * surface area. As a result, one must be known to know the other. 

Again, there is no true Pint. No in vitro assay outputs Pint. The Caco2 assay output is apparent Pint but not Pint. Why, because area is uncertain (cross sectional area is known but it is not the true area that the drug is encountering on the way through the membrane). Your idea of one side of the membrane being truly the cross sectional area is interesting but does not negate that the side with the brush border is not the cross sectional area. Perhaps that is why a three compartment structure (donor, receiver, intramembrane), as opposed to a two compartment structure of donor and receiver, better defines the movement of drug through a caco2 system to account for the intramembrane concentration differential (as compared to the donor and receiver sides) and perhaps the surface area difference between the apical and basolateral sides. I don’t know. I would say however that if the surface area doubles due to villi and microvilli, the flux from one side of a membrane to the other would double. The drug then finds itself in the cell and encounters yet another membrane to transverse with likely the same Pint but a different surface area. 

“Further, the transcellular permeability component of Caco-2 and MDCK was shown to reflect closely to 1:1 the corresponding component of human jejunal in vivo permeability when both are expressed according to permeability scale №1 (Avdeef and Tam, 2010; Sjorgren et al., 2013).” Correct me if I’m wrong, but the Avdeef and Tam paper used the caco2 values and optimized surface area. This is equivalent to setting a surface area and optimizing a Pint. Any uncertainty in the fixed number is rolled into the optimized value. The Avdeef and Tam paper then stated that the surface area was close to a reported enhancement factor. Enhancement factors are wide ranging in the literature and differ between gut segments so this doesn’t really say anything other than helping the researchers justify the model. I’m sure this model to generate effective P in the jejunum is good because it has been scaled for Caco2 input. In the Sjorgren model, they used effective P in the jejunum (in vivo measure) for all intestinal segments for prediction of fabs unless they didn’t have that number for a compound. When they didn’t have effective permeability, they used caco2 but not without scaling caco2 to effective permeability.  Even when they used the ‘gold standard’ of Peff, the predicted fabs for low permeability compounds were worse off than for high permeability compounds. For high permeability molecules, the model does great but that’s because when permeability is that high, the model is not sensitive to changes in Peff (so if Peff is not necessarily correct, it doesn’t matter to the output of fabs). So for this model, we must look at the low permeability and high solubility compounds to test if Peff is a good metric within the model. The paper had very few of these compounds so its really hard to tell if Peff is appropriate for fabs prediction within their model structure. 

“In contrast, the calculated in PK-Sim permeability values are on average ~60 times lower. The interlab variation covers 2-fold uncertainty. Where does the rest shift of 30 times come from?” To this point, I ask, lower than what? No in vitro assay is producing Pint so there is no comparison to be had here.

“Definitely I am not talking about precise values. Nevertheless these values are measured with the acceptable accuracy. E.g. Avdeef and Tam, 2010; Dahlgren et al., 2015” What is accuracy in the absence of a known intestinal permeability? 

My point to this is that Pint is an independent parameter. It is not affected by surface area. Experimentally, intestinal permeability and surface area are not uniquely identifiable. This is why, when folks model oral absorption, there is some flavour of regression in the model parametrization process that allows different kinds of experimental data to be used (e.g. Peff from jejunum, Papp from Caco-2, algorithm using phys-chem info). If we knew Pint, which we don’t, we wouldn’t need any of this. 

It would be great to have a Caco-2 scale for PK-Sim. I would use this for sure. This couldn’t reduce the interlab variability issue but it would still be very useful. Is this something you plan to do Tatiana?

Take care!!
Andrea

**Comment by TatianaAlieva** [comment]
Dear Andrea,
thank you for your answers again!

> The 2004 paper describing the PK-Sim gut model had a continuous tube structure and parameterization. The revised version is more of an ACAT-like model including the mucosa (Figure 2 in the 2011 paper). As stated in the abstract of the 2011 paper, “A training set of 111 passively absorbed drugs with reported
fractions of dose absorbed was used to optimize the semiempirical equation, which calculates
intestinal permeability coefficients.” The result of any optimization is a function of the model structure. Once Pint was optimized within the model for fraction absorbed of a bunch of compounds, those Pint values were regressed with a lipophilicity and size metric. What comes of this is a way of calculating Pint based on a compounds lipo and size. The way that Pint is then used in the equations to estimate rate of absorption in the model are typical such that Pint is multiplied by gut surface area.

In both papers (Willmann and Thelen) the total absorptive area of the intestine is identical: 71m2; transit times are almost identical. It is impossible that the transition from the continuous tube model to the compartmental representation of the intestine alone would generate change in the permeability of 10-300 times. Puzzled...... Perhaps, the authors of the papers could give a clue?...

> Again, there is no true Pint. No in vitro assay outputs Pint. The Caco2 assay output is apparent Pint but not Pint.

There are human _in vivo_ intestinal permeability values  for over 50 compounds (see collections of these values in e.g. Avdeef and Tam, 2010; Dahlgren et al., 2015) in jejunum, and for smaller number of compounds in other parts of the intestine (e.g. Lennernäs, 2014).
Human intestinal permeability values are measured and reported in terms of the smooth cylinder model. Independent estimates of the intestinal surface expansion factors (for the same intestinal segment) are fairly consistent (Watts and Illum, 1997; DeSesso and Jacobson, 2001; Schmidt and Thews “Human physiology”). Therefore it is possible to estimate human intestinal permeability in any earlier mentioned permeability scale and compare it to permeability of _in vitro_ assays and _in silico_ models.
These numbers do not let us manipulate with the permeability freely. The order of magnitude should be reasonable.

> “Further, the transcellular permeability component of Caco-2 and MDCK was shown to reflect closely to 1:1 the corresponding component of human jejunal in vivo permeability when both are expressed according to permeability scale №1 (Avdeef and Tam, 2010; Sjorgren et al., 2013).” Correct me if I’m wrong, but the Avdeef and Tam paper used the caco2 values and optimized surface area. 

As far as I know they did not optimized any surface area. Can you, please, give me a quotation where it is written?

> Enhancement factors are wide ranging in the literature and differ between gut segments so this doesn’t really say anything other than helping the researchers justify the model. 

I’ve seen only consistent values when referred to (at least) jejunum (Watts and Illum, 1997; DeSesso and Jacobson, 2001; Schmidt and Thews “Human physiology”), besides Avdeef and Tam compared Caco/MDCK permeability exclusively to the human jejunum permeability.

> In the Sjorgren model, they used effective P in the jejunum (in vivo measure) for all intestinal segments for prediction of fabs unless they didn’t have that number for a compound. When they didn’t have effective permeability, they used caco2 but not without scaling caco2 to effective permeability.

Of cause, they used a scaling factor, because jejunal permeability Peff is reported in terms of the “smooth cylinder” representation of the intestine; Pcaco, in contrast, refers to the unit cell layer surface area (see my previous posts). So, to compare jejunal permeability from in vivo studies to that of cell monolayers in vitro one need to multiply Peff by the surface expansion factor due to the circular folds and villi.

A non-trivial finding of both studies (Avdeef and Tam; Sjorgren et al.) was that the ratio between Peff jejunal and Pcaco turned out to be very close to the surface expansion factor due to the circular folds and villi in jejunum (microvilli is a common feature of enterocytes and Caco), suggesting that the permeability of the intestinal epithelium and Caco/MDCK cell layers when referred to the epithelium surface area are very close. 
Tatiana

**Comment by msevestre** [comment]
@TatianaAlieva @Aedginto 
Thanks a lot for the very interesting discussion. I never really realized how complicated the interpretation of intestinal permeability could be.

I just wanted to point out that I compared the calculated value of the specific intestinal permeability in PK-Sim with the value that would be calculated using the 2011-revision paper. The values in PK-Sim are absolutely consistent. So the good news is that there is **NO** problem with unit conversion and the value calculated by PK-Sim reflects the publication.

Drug | MW_eff	| Lipophilicity | P_int_PK-Sim (cm/s) | P_int_Paper (cm/s)
------ | --------- | -------- | ------ | -----
Caffeine	|194.00000	|0.10000|	1.69736E-08	|1.69892E-08
Caffeine	|194.00000	|0.60200|	5.3923E-08	|5.39725E-08
Paracetamol	|151.00000	|0.34000|	9.10931E-08	|9.11695E-08
Paracetamol	|151.00000	|0.77800|	2.49738E-07	|2.49948E-07
salysilic acid	|138.00000|	2.20000	|9.89523E-06|	9.90324E-06
Theophylline	|180.00000|	-0.02000|	1.80366E-08	|1.80528E-08
Theophylline	|180.00000|	0.88600	|1.45263E-07	|1.45393E-07

As a reminder the formula used in the paper is
```
Pint(MWeff ,MA) = 265.796 ×MW_eff^−4.49968× MA(cm/s) (5)
```

The formula used in PK-Sim is 
```
266 * (MWEff * 1E9) ^ ( - 4.5) * 10^LogMA * 60 *1E-1
```

The conversion factor `*60`  and `1E-1` are here to convert the value in `cm/s` into our internal unit `dm/min`

**Comment by msevestre** [comment]
@TatianaAlieva Going to close this issue. If you need more input, please do not hesitate to reopen

**Comment by StephanSchaller** [comment]
@TatianaAlieva, are you still active in this field? if so, I would be happy if you could reach out (stephan.schaller@esqlabs.com). Thanks!

**Comment by BB8-ctrl** [comment]
so have the problem solved ? .. I am also confused about how to use CaCO2 in pk-sim
